dist/index.js

```diff
@@ -694,6 +694,7 @@ function getInputs() {
     return new ActionInputs();
```

Comment:
I wasn't sure if I'm supposed to generate the dist files or not. Please let me know if I should remove it again.